.travis.yml

```diff
@@ -49,3 +49,5 @@ env:
 before_install: ./Support/Testing/Travis/before-install.py
 install:        ./Support/Testing/Travis/install.py
 script:         ./Support/Testing/Travis/script.sh
+after_success:
+  - coveralls --exclude lib --exclude tests --gcov-options '\-lp'
```

Comment:
The tests passed. Maybe it's impossible to fail in an after_success block though?